app/src/main/res/xml/preferences_behavior.xml

```diff
@@ -8,6 +8,12 @@
         android:title="@string/pref_focus_search"
         android:summary="@string/pref_focus_search_summary"
         app:iconSpaceReserved="false"/>
+    <androidx.preference.SwitchPreferenceCompat
+        android:defaultValue="false"
+        android:key="pref_minimize_on_copy"
+        android:title="@string/pref_minimize_on_copy_title"
+        android:summary="Minimize app after copying token"
```

Comment:
This should to be a translatable string as well. Also, slight modification suggestion: "Minimize the app after copying a token".